manifests/p/Peppy/Osu!/2021.828.0/Peppy.Osu!.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Peppy.Osu!
+PackageVersion: 2021.828.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://github.com/ppy/osu/releases/download/2021.828.0/install.exe
+  InstallerSha256: 874F751400592F4810EFAE4EB438A38C9BB4967F6C8F6B6E402F31A6972A6E18
+  InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: Peppy.Osu!

```

```suggestion
    SilentWithProgress: Peppy.Osu!
```
Oops, I forgot to spot this from @OfficialEsco's PR but is `Peppy.Osu!` supposed to be here?

---

Uhhh, no its not 🤦‍♂️ Guess the Powershell History messed me up